glyphmaps/FontAwesome5Free_meta.json

```diff
@@ -0,0 +1,3491 @@
+{
```

Comment:
This is a very verbose data structure, could we invert it by doing `{ "brands": ["500px"...] ... }` instead?

Reply:
Done 🙂